### PR TITLE
CSV importing

### PR DESCRIPTION
**Pull request by Floppy**
WIP currently, failing tests.

### CODE REVIEW

**Comment by Floppy** [comment]
Basic CSV loading is now fine, but need to understand and integrate Chargify, which wasn't done for memberships created via CapsuleCRM anyway. Hence thus and so.

**Comment by Floppy** [comment]
I think we can merge this after it's checked by someone - @pezholio? It doesn't handle Chargify yet, but we can add that as a secondary thing after import. There will be existing members who aren't in there as well, and a secondary pass to add them via the API will be fine.

**Comment by greenreveller** [comment]
If it helps - @jordelver created a system whereby we can add student members to Capsule and then which then migrates data to Chargify. It's his last morning with us today so could be with a chat (if you have already then ignore me!)

**Comment by Floppy** [comment]
@greenreveller aha! Yes, that sounds promising. @jordelver can we catch up after standup? Glad you're still here to ask. Could you review what I've done here as well?

File: app/controllers/uploads_controller.rb [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  def create
+    @members = []
+    csv = CSV.parse(params[:file].read, headers: true)
+    csv.each do |row|
+      @members << create_student_member_from_row(row)
+    end
+  end
+  
+  def sample
+    respond_to do |wants|
```
Comment (pezholio) [inline app/controllers/uploads_controller.rb]: Why is this nailed into the code?

File: app/views/uploads/create.html.erb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<h1>Uploads#create</h1>
+<p>Find me in app/views/uploads/create.html.erb</p>
```
Comment (pezholio) [inline app/views/uploads/create.html.erb]: Probably don't need this :smile:

File: app/controllers/uploads_controller.rb (hunk 1 above)
Comment (pezholio) [inline app/controllers/uploads_controller.rb]: Ignore me, I understand now :+1:

**Pull request merged**